aten/src/ATen/native/cpu/Activation.cpp

```diff
@@ -270,6 +270,18 @@ void GeluKernelImpl(TensorIteratorBase& it) {
       GeluMKLKernelImpl<scalar_t>(&it);
     });
   } else {
+    auto grain_size = at::internal::GRAIN_SIZE;
+// Numbers based on benchmarking.
+#ifdef C10_MOBILE
+    // Benchmarked on S8 US phone.
+    constexpr int64_t GELU_MIN_ELEMENTS_FOR_MULTI_THREADING{6144};
+#else
+    // Benchmarked on i9 8 core 16 thread machine.
```

Comment:
A longer description of benchmark methodology would be appreciated when someone wants to update this later